src/extensionShared.ts

```diff
@@ -9,19 +9,67 @@
  */
 
 import vscode from 'vscode'
-import globals from './shared/extensionGlobals'
+import globals, { initialize } from './shared/extensionGlobals'
 import { join } from 'path'
 import { Commands } from './shared/vscode/commands2'
 import { documentationUrl, githubCreateIssueUrl, githubUrl } from './shared/constants'
 import { getIdeProperties, aboutToolkit } from './shared/extensionUtilities'
 import { telemetry } from './shared/telemetry/telemetry'
 import { openUrl } from './shared/utilities/vsCodeUtils'
 
-export function initializeManifestPaths(extensionContext: vscode.ExtensionContext) {
-    globals.manifestPaths.endpoints = extensionContext.asAbsolutePath(join('resources', 'endpoints.json'))
-    globals.manifestPaths.lambdaSampleRequests = extensionContext.asAbsolutePath(
+import { activate as activateLogger } from './shared/logger/activation'
+import { initializeComputeRegion } from './shared/extensionUtilities'
+import { activate as activateTelemetry } from './shared/telemetry/activation'
+import { DefaultAwsContext } from './shared/awsContext'
+import { Settings } from './shared/settings'
+import { DefaultAWSClientBuilder } from './shared/awsClientBuilder'
+import { initialize as initializeAuth } from './auth/activation'
+import { LoginManager } from './auth/deprecated/loginManager'
+import { CredentialsStore } from './auth/credentials/store'
+import { initializeAwsCredentialsStatusBarItem } from './auth/ui/statusBarItem'
+import { RegionProvider } from './shared/regions/regionProvider'
+
+/**
+ * This runs activation/setup code that is shared by both the regular extension vs. browser
+ * activation in their respective entrypoints.
+ *
+ * The different entrypoints may do additional setup, but the setup in this function
+ * is shared by both and works for both. So any code that works in both should be
+ * placed here.
+ *
```

Comment:
This is very verbose.
```suggestion
 * Activation/setup code that is shared by the regular (nodejs) extension and
 * the browser-compatible extension. Most setup code should live here.
 *
```